be/src/http/action/transaction_stream_load.cpp

```diff
@@ -102,10 +102,20 @@ void TransactionManagerAction::handle(HttpRequest* req) {
         return _send_error_reply(req, Status::InvalidArgument(fmt::format("empty label")));
     }
 
+    bool httpHeaderContainChannelId = false;
+    int channel_id;
+    if (!req->header(HTTP_CHANNEL_ID).empty()) {
+        httpHeaderContainChannelId = true;
+        channel_id = std::stoi(req->header(HTTP_CHANNEL_ID));
```

Comment:
What's the difference between this with prepare and commit interfaces?

Reply:
the orignal transaction stream load's max parallelism is 1